dist/src/main/package/www/services/common_services.js

```diff
@@ -349,6 +349,7 @@ myControllerModule.factory('CommonServices', function(TypesFactory, $filter, $co
         {id:"Heater", value:"fa fa-eye", ucode:"\uf06e", fname:"FontAwesome"},
         {id:"Distance", value:"fa fa-binoculars", ucode:"\uf1e5", fname:"FontAwesome"},
         {id:"Light level", value:"wi wi-moon-alt-waxing-crescent-5", ucode:"\uf0d4", fname:"weathericons"},
+        {id:"Hue light", value:"wi wi-moon-alt-waxing-crescent-5", ucode:"\uf0d4", fname:"weathericons"},
```

Comment:
@Fraid I believe, we are not using these `Hue light` in anywhere, we can remove this